proxy/cypress/integration/login/login.spec.js

```diff
@@ -0,0 +1,33 @@
+context("Login page", () => {
+  beforeEach(() => {
+    cy.visit("/MAAS/r");
+  });
+
+  it("gets redirected to correctly", () => {
```

Comment:
Was this meant to be "gets redirected correctly"?

Reply:
maybe >_>